BeeTracker/app/src/main/res/values/strings.xml

```diff
@@ -2,4 +2,7 @@
     <string name="app_name">Bee Tracker</string>
     <string name="facebook_app_id">221233091726222</string>
     <string name="fb_login_protocol_scheme">fb221233091726222</string>
+    <string name="SubmitButton">Submit</string>
+    <string name="title_activity_submission">SubmissionActivity</string>
```

Comment:
Do you use these strings anywhere? All strings should be put in this file, but they should also be used